cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ template "crossplane.name" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
@AaronME does providing it explicitly change behavior from what currently happens with `helm install -n <namespace>`?

Reply:
The expected behavior is the same for `helm install -n <namespace>`

This change makes it possible to run `helm template -n <namespace> <name> <chart>` and get manifests with explicit namespaces declared. These can then be piped through other tools before applying them to the cluster. (Some checkers have flagged the lack of a namespace as "needing attention.")

Reply:
Nice! Thanks for the explanation 👍🏻